scripts/lighthouse/logResults.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+const chalk = require('chalk'); // eslint-disable-line import/no-extraneous-dependencies
+
+function formatResult({ id, score, expectedScore, pass }) {
+  const resultDetail = `${id}, actual: ${score}, expected: ${expectedScore}`;
+  if (pass) {
+    console.log(`${chalk.black.bgGreen(' PASS ')} ${resultDetail}`);
+  } else {
+    console.log(`${chalk.black.bgRed(' FAIL ')} ${resultDetail}`);
+  }
+}
+
+function logHighLevelScores(results) {
+  const failures = [];
+  results.forEach(result => {
+    console.log(chalk.underline(`\nLighthouse results for ${result.url}:`));
+    result.scores.forEach(score => {
+      formatResult(score);
```

Comment:
This isn't so much 'formatting' the result as it is logging it, but its very picky of me